docs/api/rest_api/rest_api_reference/input/ez-types.raml

```diff
@@ -2128,401 +2140,1056 @@ View: #TODO outdated example
 
 ProductVariantGenerate:
+    description: 'This class is used to generate a product variant.'
     type: object
-    description: ''
+    properties:
+        attributes:
+            type: array<int|string,mixed>
 
 ProductVariantUpdate:
+    description: 'This class is used to update a product variant.'
     type: object
-    description: ''
+    properties:
+        code:
+            type: string
+        attributes:
+            type: array<int|string,mixed>
 
 ProductVariantViewInput:
+    description: 'This class represents a product variant view input.'
     type: object
-    description: ''
+    properties:
+        ProductVariantQuery:
+            type: object
 
 ProductVariantView:
+    description: 'This class represents a product variant view.'
     type: object
+    properties:
+        identifier:
+            type: string
+        ProductVariantQuery:
+            type: object
+        Result:
+            type: object
 
 RegionViewInput:
+    description: 'This class represents a region view input.'
     type: object
-    description: ''
+    properties:
+        RegionQuery:
+            type: object
 
 RegionView:
+    description: 'This class represents a region view.'
     type: object
+    properties:
+        identifier:
+            type: string
+        RegionQuery:
+            type: object
+        Result:
+            type: object
 
 ProductTypeList:
+    description: 'This class represents a product type list.'
     type: object
-    description:
+    properties:
+        ProductTypes:
+            type: ProductType[]
 
 TaxonomyEntry:
-    type: object
+    description: 'This class represents a taxonomy entry.'
+    type: BaseObject
+    properties:
+        id:
+            type: integer
+        identifier:
+            type: string
+        name:
+            type: string
+        mainLanguageCode:
+            type: string
+        names:
+            type: array
+        Parent:
+            type: Parent
+        Content:
+            type: Content
+        taxonomy:
+            type: string
+        level:
+            type: integer
 
 TaxonomyEntryAssignment:
+    description: 'This class represents a taxonomy entry assignment.'
     type: object
+    properties:
+        id:
+            type: integer
+        Content:
+            type: Content
+        Entry:
+            type: Entry
 
 EntryAssignmentCollection:
+    description: 'This class represents a taxonomy assignment collection.'
     type: object
+    properties:
+        Content:
+            type: Content
+        assignments
+            type: TaxonomyEntryAssignment[]
+
+TaxonomyEntryCreateStruct:
+    description: 'This class is used to create a taxonomy entry.'
+    type: BaseObject
+    properties:
+        identifier:
+            type: string
+        taxonomy:
+            type: string
+        Parent:
+            type: Parent
+        Content:
+            type: Content
+
+TaxonomyEntryUpdateStruct
```

Comment:
`Unable to parse at line 3127 (near "TaxonomyEntryUpdateStruct").`

Missing colon:
```suggestion
TaxonomyEntryUpdateStruct:
```